lib/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -335,4 +335,9 @@ public boolean enabled(FeatureFlag featureFlag) {
         return Boolean.parseBoolean(
                 getSsmParameter(ConfigurationVariable.FEATURE_FLAGS, featureFlag.getName()));
     }
+
+    public boolean enabled(String featureFlagValue) {
```

Comment:
Can we add a test to the ConfigServiceTest to cover this, just to keep that code covereage up :)  otherwise LGTM

Reply:
Good shout, done